updated -h command line output

DerekCuevas opened the pull request:
Attempting to fix issue #149. Not sure if this is exactly what you meant. Is this what you had in mind?

mfikes commented:
Hi @DerekCuevas. Thanks! Actually I had in mind just putting in equals signs in the right places. So, something like

```
    -i, --init=path     Load a file or resource
    -e, --eval=string   Evaluate expressions in string; print non-nil values
    -c, --classpath=cp  Use colon-delimited cp for source directories and JARs
```

or maybe even

```
    -i path, --init=path        Load a file or resource
    -e string, --eval=string Evaluate expressions in string; print non-nil values
    -c cp, --classpath=cp   Use colon-delimited cp for source directories and JARs
```

if that is clearer.

Also:

1. There is an integration test that will look for the expected output that would need to be revised.
2. Looks like you committed a change to `PLKBundledOut.m` which you'd want to revert and squash out of the commit.

DerekCuevas commented:
Ok got it. Thanks for the feedback. I'll try to revise my pull request with the correct changes.

DerekCuevas commented:
How does this look? I reverted the bad commit, edited all options that take arguments, and updated the integration tests. Also the lines do not go over 80 characters.

mfikes commented:
Can these be squashed? Otherwise there's a gigantic set of 180,000 additions followed by deletions. :)

DerekCuevas commented:
Sorry! Should be fixed now.

mfikes commented:
Thanks!
